app/services/foreman/parameters/validator.rb

```diff
@@ -13,6 +13,10 @@ def validate!
           validate_regexp
         when "list", "array"
           validate_list
+        when "hash"
```

Comment:
reproduction:
select "hash" as type of value
input { "foo" => "bar } into default value (or lookup values)
see everything crash and burn